src/lib/src/footer/footer.component.css

```diff
@@ -2,12 +2,10 @@
  * PRX footer component
  */
 footer {
-  background: #fff;
+  background: #1c1c1c;
+  color: #bebebe;
```

Comment:
Not sure on medium grey tone either... There is some kind of sorcery on the square space site that makes it look like these should be inheriting white from the footer, but they look grey to me.